app/src/main/java/com/berniesanders/connect/MainActivity.java

```diff
@@ -32,8 +48,21 @@ protected void onCreate(Bundle savedInstanceState) {
         mText.setText(mContext.getPackageName());
 
         mActionAlertsModel.getActionAlerts().subscribe(
-                alerts -> Timber.d("alerts: " + alerts),
-                error -> {},
-                () -> {});
+                alerts -> mAlertListView.setAdapter(new ActionAlertAdapter(mContext, alerts)),
```

Comment:
You can blame me. I was using it as assurance the the Dagger 2 code was actually working when I was first implementing it. The context can be removed.